sse2neon.h

```diff
@@ -8358,6 +8359,638 @@ FORCE_INLINE int _mm_testz_si128(__m128i a, __m128i b)
 
 /* SSE4.2 */
 
+typedef enum {
+    _MM_CMPINT_EQ = 0,    /* Equal */
+    _MM_CMPINT_LT = 1,    /* Less than */
+    _MM_CMPINT_LE = 2,    /* Less than or Equal */
+    _MM_CMPINT_FALSE = 3, /* Always False */
+    _MM_CMPINT_NE = 4,    /* Not Equal */
+    _MM_CMPINT_NLT = 5,   /* Not Less than */
+    _MM_CMPINT_NLE = 6,   /* Not Less than or Equal */
+    _MM_CMPINT_TRUE = 7   /* Always True */
+} _MM_CMPINT_ENUM;
+
+static uint16_t g_mask_epi16[8] __attribute__((aligned(16))) = {
```

Comment:
Both `g_mask_epi16` and `g_mask_epi8` are not meaningful variable names. Can you improve?